airflow/contrib/example_dags/example_gcp_compute_igm.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that Uses IGM-type compute operations:
+* copy of Instance Template
+* update of the group manager's templates
+
+This DAG relies on the following Airflow variables
+https://airflow.apache.org/concepts.html#variables
+* PROJECT_ID - Google Cloud Platform project where the Compute Engine instance exists.
+* ZONE - Zone where the Compute Engine instance exists.
+* TEMPLATE_NAME - Name of the template to copy.
+* NEW_TEMPLATE_NAME - name of the new template.
+* NEW_DESCRIPTION - description added to the template
+* SOURCE_TEMPLATE_URL - url of the template to replace
```

Comment:
Is the source being replaced? Or being copied?

Reply:
Right :)